player/scripting.c

```diff
@@ -254,7 +254,8 @@ typedef int (*mpv_open_cplugin)(mpv_handle *handle);
 
 static int load_cplugin(struct mpv_handle *client, const char *fname)
 {
-    int r = -1;
+    char *err;
```

Comment:
Move the decl. to where it's used.

Reply:
```
../player/scripting.c:269:5: error: a label can only be part of a statement and a declaration is not a statement
     char *err = dlerror();
     ^~~~
```
I've tried, trust me, but gcc won't let me.

Reply:
That's just C being dumb and not allowing a label before declaration. Can be worked around by adding a `;` after the label, like `end: ;`.